app/views/AnalyticalFramework/FrameworkForm/Review/index.tsx

```diff
@@ -11,7 +11,7 @@ import { _cs } from '@togglecorp/fujs';
 import { Section, Widget } from '#types/newAnalyticalFramework';
 import _ts from '#ts';
 
-import Canvas from '../components/Canvas';
+import Canvas from '#views/Project/EntryEdit/components/Canvas';
```

Comment:
Let's add FIXME to move the Canvas component outside views

Reply:
Done